pyneuroml/neuron/analysis/HHanalyse.py

```diff
@@ -535,6 +535,14 @@ def main():
             file.close()
             print("Written info to file: %s" % file_name)
 
+    figs = pylab.get_fignums()
+
+    for fig in figs:
```

Comment:
Yeah, I suppose that makes more sense. 
But this scenario is for, if we want to save the figure as well as see the figure which might be useful in case of bulk processing.
Let me know if I should keep it or change it to only save the figure in case of `-nogui`
or maybe we can introduce another argument which will allow both saving and showing the figure

Reply:
Yeh, maybe we add a boolean "save_plots" argument which can be enabled by default but modifiable as command line argument too?

The use cases here are:

- I analyse a single mod file and I want to see the plots, and optionally save them: `nogui=False, save_plots=True/False`
- I analyse lots of channels, and I don't want to see plots: `nogui=True, save_plots=True`

Reply:
Yeah I can do that. Shall I go ahead ?
I have tried locally and its working

Reply:
@sanjayankur31 Done! PTAL